pyswervedrive/swervemodule.py

```diff
@@ -169,20 +172,19 @@ def set_velocity(self, vx, vy):
         :param vy: desired y velocity, m/s (y is left on the robot)
         """
 
+        if math.hypot(self.vx, self.vy) != 0 and math.hypot(vx, vy) == 0:
+            self.drive_motor.setIntegralAccumulator(0, 0, 10)
```

Comment:
I just realised, shouldn't this call also be a fire-and-forget sort of thing?

Reply:
I guess, yeah.